rtt.go

```diff
@@ -0,0 +1,52 @@
+package nex
+
+import (
+	"math"
+	"sync"
+	"time"
+)
+
+const (
+	Alpha float64 = 1.0 / 8.0
+	Beta  float64 = 1.0 / 4.0
+	K     float64 = 4.0
+)
+
+// RTT is an implementation of rdv::RTT and is used to calculate the average round trip time of reliable packets
```

Comment:
Just to split this up, for Godoc purposes

```suggestion
// RTT is an implementation of rdv::RTT.
// Used to calculate the average round trip time of reliable packets
```